docs/source/notebooks/00_Introduction_to_JupyterLab.ipynb

```diff
@@ -0,0 +1,242 @@
+{
```

Comment:
<p>Add link to "PAVICS-Hydro documentation"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/raven/pull/424/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Ideally yes, but I would need somebody to put the file on the website and send me the link to it.</p>

Reply:
@huard @tlogan2000 The document is here, is it possible to host it somewhere so we can link to it in the notebooks?

[PAVICS-Hydro Tutorial.pdf](https://github.com/Ouranosinc/raven/files/8985726/PAVICS-Hydro.Tutorial.pdf)

thanks!